### ISSUE TITLE
[3.4.1] Crashes at startup. 

### ISSUE THREAD

**Issue by joshparrish** [Ultimaker/Cura#4239]
Crashes on startup. Splash screen is shown and it loads until "Loading Interface"
**Application Version**
 
3.4.1
**Platform**
 Windows 10 

**Additional Information**

[cura.log](https://github.com/Ultimaker/Cura/files/2291096/cura.log)

**Comment by joshparrish** [comment]
[DxDiag.txt](https://github.com/Ultimaker/Cura/files/2291673/DxDiag.txt)

**Comment by LipuFei** [comment]
Hi @joshparrish , sorry for the late reply. Could you upload your log file? You can find it in
```
C:\Users\<your-user-name>\AppData\Roaming\cura\<cura-version>\cura.log
```

**Comment by Ghostkeeper** [comment]
This is fixed in 4.0. The entire component that gave the error doesn't exist any more due to the redesign.
